packages/node-integration-tests/suites/tracing/prisma-orm/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+NODE_MAJOR=$(node -v | cut -c2- | cut -d. -f1)
+
+if [ "$NODE_MAJOR" -lt "12" ]; then
+  echo "Skipping Prisma tests on Node: $NODE_MAJOR"
```

Comment:
I think going forward we would like everything to be node scripts. Can we convert this to a TS script (we have `ts-node` in the repo)?